GliaWidgetsObjC/README.md

```diff
@@ -0,0 +1,23 @@
+## Objective-C wrapper for [GliaWidgets](https://github.com/salemove/ios-sdk-widgets)
+
+GliaWidgetsSDK developed used pure Swift and doesn't provide Objective-C compatible interfaces. This sample Xcode project shows possibility to get full compatibility with Objective-C.
+
+##### NB! This project is not updating with latest released GliaWidgetsSDK versions and shows how to get Objective-C interfaces only.
+
+### Project
+
+Xcode project nests all GliaWidgetsSDK frameworks, introduces `GliaWidgetsWrapper.swift` that maps Swift interfaces to Objective-C.
+
+### How does it work?
+The `@objc` and `@objcMembers` attributes in Swift make Swift API available in Objective-C and the Objective-C runtime.
+
+### How to use
+
+To import and use `ios-sdk-widgets-objc` framework you need to:
+- update nested frameworks to latest versions. [See "Update Dependency"](#update-dependency)
```

Comment:
Making it a numbered list because I assume it's best to do these things in this order:
```suggestion
1. Update the nested frameworks to latest versions. See ["Updating Dependencies"](#updating-dependencies) for guidelines.
```